doc/whats_new/upcoming_changes/sklearn.feature_selection/31483.feature.rst

```diff
@@ -0,0 +1,2 @@
+- :class:`feature_selection.SequentialFeatureSelector` now stores the final cross-validation score of the selected feature set in a new attribute :attr:`final_cv_score_`.
```

Comment:
This line seems a bit long considering the repository's current standards. Could you split it up a bit?